docs/reference-guides/filters/editor-filters.md

```diff
@@ -1,23 +1,166 @@
 # Editor Hooks
 
-To modify the behavior of the editor experience, WordPress exposes several APIs.
+WordPress exposes several APIs that allow you to modify the editor experience.
+
+## Editor settings
+
+One of the most common ways to modify the Editor is through the [`block_editor_settings_all`](https://developer.wordpress.org/reference/hooks/block_editor_settings_all/) PHP filter, which is applied before settings are sent to the initialized Editor. This filter allows plugin and theme authors extensive control over how the Editor behaves.
+
+<div class="callout callout-warning">
+	Before WordPress 5.8, this hook was known as <code>block_editor_settings</code>, which is now deprecated. If you need to support older versions of WordPress, you might need a way to detect which filter should be used. The recommended way to proceed is to check if the <code>WP_Block_Editor_Context</code> class exists.
```

Comment:
> The recommended way to proceed is to check if the <code>WP_Block_Editor_Context</code> class exists.
In which way checking if the <code>WP_Block_Editor_Context</code> class exists, does allow to determine which filter should be used? If it does exist then [block_editor_settings_all](https://developer.wordpress.org/reference/hooks/block_editor_settings_all/) should be used? 

Maybe we could be more clear about this here.

Reply:
Good call, I'll update. `WP_Block_Editor_Context` was introduced in 5.8, so it exists, you are safe to use `block_editor_settings_all`.